.github/workflows/all.yml

```diff
@@ -135,17 +135,36 @@ jobs:
     outputs:
       version: ${{ steps.get_version.outputs.version }}
     steps:
+      - name: Checkout code
+        if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
maybe this should happen if it *doesn't* start with this ;)

Reply:
yeah, noticed it and :man_facepalming:-ed :sweat_smile: